src/modules/fancyzones/FancyZonesTests/UnitTests/AppZoneHistoryTests.Spec.cpp

```diff
@@ -1,7 +1,13 @@
 #include "pch.h"
 #include <filesystem>
 
+// disabling warning 4702 - unreachable code
+// prevent the warning on functions not used in tests.
+#pragma warning(push)
+#pragma warning(disable : 4702)
 #include <FancyZonesLib/FancyZonesData/AppZoneHistory.h>
+#pragma warning(pop)
+
```

Comment:
I think it would be better to update AppZoneHistory.h.
If you add `#else` in `AppZoneHistoryFileName` disabling wouldn't be necessary.

```
#if defined(UNIT_TESTS)
        return ...;
#else
        return ...;
#endif
```

The same for others, such as AppliedLayout, CustomLayouts, LayoutHotkeys, etc.

Reply:
good, it will reduce the number of #pragma =)